components/Layout.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  ChevronRightIcon,
+  ArrowTopRightOnSquareIcon,
+  HomeIcon,
+  Bars3Icon,
+  NewspaperIcon,
+  UsersIcon,
+} from '@heroicons/react/24/outline'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import Script from 'next/script'
+import { useState } from 'react'
+import Logo from '../public/logo.svg'
+
+type Indexable = {
+  [key: string]: any
+}
+
+const navigation = [
+  {
+    name: 'Citizen directory',
+    icon: <UsersIcon className="h-5 w-5" />,
+    href: '/',
+  },
+  {
+    name: 'Homepage',
+    href: 'https://nation3.org',
+    icon: <HomeIcon className="h-5 w-5" />,
+  },
+  {
+    name: 'Docs',
+    href: 'https://docs.nation3.org/',
+    icon: <NewspaperIcon className="h-5 w-5" />,
+  },
+]
```

Comment:
We should give this a type, and make icon an Icon, applying the style inside the component.